[NFS] docs: update nfs status (#30959)
backstage/backstage: docs/frontend-system/index.md

## Status

> We recommend migrating any apps to the new system. If you add support for the new system to your plugin, please do so under a `/alpha` sub-path export.

Reviewer: ```suggestion
We recommend migrating your frontend plugins to the new frontend system. If you do please do so under an `/alpha` sub-path export.
```

Yours is also fine I think, more just a suggestion 😄
